pallets/eth-bridge/src/lib.rs

```diff
@@ -229,21 +266,64 @@ pub mod pallet {
             _signature: <T::AuthorityId as RuntimeAppPublic>::Signature,
         ) -> DispatchResultWithPostInfo {
             ensure_none(origin)?;
-            util::update_confirmations::<T>(tx_id, &confirmation, &author)?;
+
+            if tx::is_active::<T>(&tx_id) {
```

Comment:
This happening wouldn't be an error or even unexpected, it's completely valid and expected that it will occur. This is because we finalize a transaction (and remove it as the "active" transaction) immediately upon receiving the first confirmation that fulfils the quorum. So we will very likely get "add_confirmation" extrinsics arriving after the transaction has been finalized.